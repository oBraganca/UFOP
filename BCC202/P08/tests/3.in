1
23
mariana 25
pedro 31
isabella 27
gabriel 18
lucas 34
ana 29
gustavo 33
andre 25
sophia 26
joao 30
carolina 28
rafael 32
lara 24
bruno 36
alice 23
andre 37
gabriel 35
laura 31
miguel 26
maria 33
arthur 27
beatriz 29
andre 30