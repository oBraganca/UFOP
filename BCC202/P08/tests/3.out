Teste 1: 
alice - 23
ana - 29
andre - 25
andre - 30
andre - 37
arthur - 27
beatriz - 29
bruno - 36
carolina - 28
gabriel - 18
gabriel - 35
gustavo - 33
isabella - 27
joao - 30
lara - 24
laura - 31
lucas - 34
maria - 33
mariana - 25
miguel - 26
pedro - 31
rafael - 32
sophia - 26